### PR TITLE
[3006.x] Add mako to pkg dependencies

### PR DESCRIPTION
**Pull request by dwoz**
### What does this PR do?

See title.

### What issues does this PR fix or reference?
Fixes: #66091

### Merge requirements satisfied?
**[NOTICE] Bug fixes or features added to Salt require tests.**

- [ ] Docs
- [x] Changelog - https://docs.saltproject.io/en/master/topics/development/changelog.html
- [ ] Tests written/updated

### CODE REVIEW

**Comment by s0undt3ch** [comment]
`mako` can be `salt-pip` installed at any time.

If we add `mako` why would we deny someone requesting `cheetah3`, `genshi` and `wempy`?

**Comment by s0undt3ch** [comment]
Even with the old packages, the user reporting the issue had to somehow install mako to use mako templates...

File: requirements/static/pkg/windows.in [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # This file only exists to trigger the right static compiled requirements destination
 # Don't add any requirements here, add them in requirements/windows.txt
+mako
```
Comment (twangboy) [inline requirements/static/pkg/windows.in:3]: I love how you ignore the comment right above this...

**Review by twangboy — CHANGES_REQUESTED** [review]

**Comment by dwoz** [comment]
The mako renderer has been moved to core extensions

**Pull request closed without merging**